### PR TITLE
Update function.dd - Return Scope parameters section

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

**Comment by thewilsonator** [comment]
Sigh
```
Searching for trailing whitespace
./spec/function.dd:1548: 
```

File: spec/function.dd [hunk 1]
```diff
@@ -1542,6 +1546,9 @@ class C
         $(P Template functions, auto functions, nested functions and lambdas can deduce
         the `return scope` attribute.)
 
+        $(P $(B Note:) Checks for `scope` parameters are currently enabled
```
Comment (ZombineDev) [inline spec/function.dd]: `return` as well

**Comment by andralex** [comment]
@ZombineDev please make a small PR to amend this - optimizing for bandwidth!!

**Comment by wilzbach** [comment]
Hmm, imho it's _not_ job of the reviewers to fix PRs or invest additional work if someone merged the PR _too early_ :/

> optimizing for bandwidth

This will undoubtedly result in __lower quality__ if we keep merging PRs so incredibly fast and ignore _existing reviews_ on them...

**Pull request merged**